doc/website/getting-started/installation.md

```diff
@@ -1,23 +1,23 @@
 # Installation
-
-_iceoryx_utils_ and _iceoryx_posh_ are deployed as independent CMake packages. _iceoryx_posh_ is using some functions from _iceoryx_utils_ and is depending on it.
+All iceoryx libraries are deployed as independent CMake packages. Posh is using functions from utils and is depending on it. You are able to build posh and utils and integrate them into existing CMake projects.
 
 ## Prerequisites
 
 ### Dependencies
 
-- 64-bit hardware (e.g. x86_64 or aarch64; 32-bit hardware might work, but is not supported)
-- [CMake](https://cmake.org), 3.5 or later
-- One of the following compilers:
-  - [GCC](https://gcc.gnu.org), 7.4 or later (5.4 currently supported too)
-  - [Clang](https://clang.llvm.org), 9.0 or later
-  - [MSVC](https://visualstudio.microsoft.com/de/), part of Visual Studio 2019 or later
-- [libacl](http://download.savannah.gnu.org/releases/acl/), 2.2 or later. Only for Linux & QNX.
-- optional, [ncurses](https://invisible-island.net/ncurses/), 6.2 or later. Required by introspection tool.
+ - 64-bit hardware (e.g. x86_64 or aarch64; 32-bit hardware might work, but is not supported)
+ - [CMake](https://cmake.org), 3.10 or later
+ - One of the following compilers:
+   - [GCC](https://gcc.gnu.org), 7.4 or later (5.4 currently supported too)
+   - [Clang](https://clang.llvm.org), 9.0 or later
+   - [MSVC](https://visualstudio.microsoft.com/de/), part of Visual Studio 2019 or later
```

Comment:
This is displayed on the same level as the previous bullet points. Maybe try `*`?